packages/block-editor/src/components/inserter-list-item/style.scss

```diff
@@ -30,7 +30,26 @@
 
 	&:not(:disabled) {
 		&:hover {
-			color: var(--wp-admin-theme-color) !important;
+			.block-editor-block-types-list__item-title {
+				color: var(--wp-admin-theme-color) !important;
+				filter: brightness(0.95);
+			}
+
+			svg {
+				color: var(--wp-admin-theme-color) !important;
+			}
+
+			&::after {
+				content: "";
+				position: absolute;
+				top: 0;
+				bottom: 0;
+				left: 0;
+				right: 0;
+				border-radius: $radius-block-ui;
+				opacity: 0.04;
+				background: var(--wp-admin-theme-color);
```

Comment:
I think this breaks dragging-and-dropping from the inserter in Firefox. (Took me a while to find this! For some reason git blame failed me 😞.)

A simple solution to this would be adding `pointer-events: none` to the `::after` pseudo element.